### PR TITLE
Add 'type' attribute to Sequelize adapter

### PR DESCRIPTION
**Pull request by actorius**
Attribute `'type'` always is `'objects'`, this PR represent `'type'` to Sequelize.Model.getTableName().
P.S. Check coffee code, i'm only test it on plain JS.

### CODE REVIEW

File: src/yayson/adapters/sequelize.coffee [hunk 1]
```diff
@@ -3,5 +3,6 @@ Adapter = require '../adapter'
 class SequelizeAdapter extends Adapter
   @get: (model, key) ->
     model.get(key) if model?
-
+  @type: (model) ->
+    return model.Model.getTableName();
```
Comment (jede) [inline src/yayson/adapters/sequelize.coffee]: No semi colons :)

**Comment by jede** [comment]
Thanks! Looks good, do you think you can add a test for it or update the tests so that they check for type?

**Comment by actorius** [comment]
My appologies for semocolons =)  i'm not pretty know coffee and for the same reason i can not write test on coffee =(

**Comment by jede** [comment]
Hm, I'm thinking about this a bit more. When would you need to have dynamic types? Right now you specify the type as a string in the presenter?

**Comment by LogansUA** [comment]
:+1:

**Comment by jede** [comment]
I'm closing this for now until I know the use case for dynamic types.

**Pull request closed without merging**